launch/base_hardware.launch

```diff
@@ -5,6 +5,10 @@
     <!-- if we're in the sim -->
     <group if="$(eval str(arg('platform')) == 'sim')" >
         <include file="$(find iarc7_simulator)/launch/morse.launch" />
+
+        <include file="$(find iarc7_vision)/launch/vision_node.launch">
```

Comment:
This really shouldn't be in base_hardware

Reply:
Yeah it's not hardware.